src/ontology/mondo-ingest.Makefile

```diff
@@ -172,6 +172,7 @@ update-jinja-sparql-queries: python-install-dependencies
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# also creates: config/%_exclusion_reasons.robot.template.tsv
```

Comment:
This feels like a code smell; the target name represents one file, but two files are generated (it generates a `.txt` and a `.robot.template.tsv`). Not important enough to worry about right now, unless @matentzn knows a better way to do this.

Reply:
Done!

Reply:
@matentzn This is kind of minor, but I'm wondering if, in the other PR, I should remove `--outpath`. Notice the difference below. Which pattern is better?

Exclusion tables feature. 2 files produced. No `outpath` param.
```make
config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: <dependencies>
	python3 $(SCRIPTSDIR)/exclusion_term_expansion.py \
	--onto-name $* \
	--exclusions-path $(word 1,$^) \
	--onto-path $(TMPDIR)/$(word 2,$^).owl \
	--mirror-signature-path $(word 3,$^) \
	--component-signature-path $(word 4,$^) \
	--config-path $(word 5,$^)
```

Excluded terms in mondo. 2 files produced. Has `outpath` param.
```make
$(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_in_mondo.tsv: $(TMPDIR)/mondo.sssom.tsv tmp/mondo.owl metadata/%.yml $(REPORTDIR)/component_signature-%.tsv $(REPORTDIR)/mirror_signature-%.tsv python-install-dependencies
	python3 ../analysis/problematic_exclusions.py \
	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
	--mondo-path tmp/mondo.owl \
	--onto-name $* \
	--onto-path $(TMPDIR)/component-download-$*.owl.owl \
	--onto-config-path metadata/$*.yml \
	--mirror-signature-path $(REPORTDIR)/mirror_signature-$*.tsv \
	--component-signature-path $(REPORTDIR)/component_signature-$*.tsv \
	--outpath $@
```